packages/mobx-react-lite/__tests__/useLocalObservable.test.tsx

```diff
@@ -440,75 +439,108 @@ describe("enforcing actions", () => {
         mobx.configure({ enforceActions: "never" })
         consoleWarnMock = jest.spyOn(console, "warn").mockImplementation(() => {})
 
-        const { result } = renderHook(() => {
-            const [multiplier, setMultiplier] = React.useState(2)
-            const store = useLocalObservable(() => ({
-                multiplier,
-                count: 10,
-                get multiplied() {
-                    return this.multiplier * this.count
-                },
-                inc() {
-                    this.count += 1
+        const onError = jest.fn()
+        const { result } = renderHook(
+            () => {
+                const [multiplier, setMultiplier] = React.useState(2)
+                const store = useLocalObservable(() => ({
+                    multiplier,
+                    count: 10,
+                    get multiplied() {
+                        return this.multiplier * this.count
+                    },
+                    inc() {
+                        this.count += 1
+                    }
+                }))
+                useEffect(() => {
+                    store.multiplier = multiplier
+                }, [multiplier])
+                useEffect(() => setMultiplier(3), [])
+            },
+            {
```

Comment:
NIT: This pattern reoccures quite a few times in the test, might be nice to make a utility for it? (separate PR is fine if interested)